website/docs/usages/mutations.mdx

```diff
@@ -9,7 +9,9 @@ Mutations methods are exposed in a straightforward way from the generated KitQL
 
 ```svelte
 <script lang="ts">
-	function add(id: string) {
+	// import { KQL_AddStar } from '$lib/graphql/_kitql/graphqlStores';
```

Comment:
I think adding this to more pages is also useful. Helps me quickly look for the files I should be importing.